### PR TITLE
Add post-start check for logstash ingestors using syslog.

### PR DESCRIPTION
**Pull request by bengerman13**
We're using these post-checks essentially to fail a deployment if it leaves the cluster in a bad state, and thought others might like to make use of them. We have them defaulted to on, but I'd be happy to reverse the default if that helps get this merged.

### CODE REVIEW

**Comment by bengerman13** [comment]
I signed the CLA, but it still shows as needing to be signed. Does that mean @wjwoodson needs to sign it, too?

**Comment by bengerman13** [comment]
@wjwoodson I think we confirmed in another PR that the issue here is we need you to sign the CLA if you're willing.

File: jobs/archiver_syslog/spec [hunk 1]
```diff
@@ -90,6 +91,16 @@ properties:
     description: Transport protocol to use
     default: "tcp"
 
+  logstash_ingestor.health.disable_post_start:
+    description: Allow node to run post-start script? (true / false)
+    default: false
```
Comment (axelaris) [inline jobs/archiver_syslog/spec]: There are some inconsistency between property name and description.
`Disable = false` should mean `enable`. However, `Allow = false` should mean `disable`.
Could you please elaborate that?

File: jobs/ingestor_syslog/spec [hunk 2]
```diff
@@ -98,6 +99,16 @@ properties:
   logstash_ingestor.syslog.use_keepalive:
     description: Instruct the socket to use TCP keep alives
 
+  logstash_ingestor.health.disable_post_start:
+    description: Allow node to run post-start script? (true / false)
+    default: false
```
Comment (axelaris) [inline jobs/ingestor_syslog/spec]: There are some inconsistency between property name and description.
`Disable = false` should mean `enable`. However, `Allow = false` should mean `disable`.
Could you please elaborate that?

**Comment by axelaris** [comment]
Hi @bengerman13, thank you for contribution! Could you please check my comments in the code?

File: jobs/archiver_syslog/spec (hunk 1 above)
Comment (bengerman13) [reply jobs/archiver_syslog/spec]: The shorter path for my team would be to update the description to something like `Skip the post-start script` - does that work for you

**Comment by bengerman13** [comment]
Thanks for looking @axelaris. I responded to one of the comments.

I think we're stuck until @wjwoodson can sign the CLA, so no rush here :)

File: jobs/archiver_syslog/spec (hunk 1 above)
Comment (axelaris) [reply jobs/archiver_syslog/spec]: Yes please

**Comment by axelaris** [comment]
We disabled CLA eventually, so no worries.
Could you please also rebase your PR to include bpm?

**Comment by siennathesane** [comment]
@bengerman13 *bump*

File: jobs/archiver_syslog/spec (hunk 1 above)
Comment (bengerman13) [reply jobs/archiver_syslog/spec]: Sorry for the lag - this has been updated now

**Comment by axelaris** [comment]
LGTM

**Comment by axelaris** [comment]
Thank you @bengerman13 !

**Pull request merged**